vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -43,7 +43,7 @@ packages:
         - libssl-dev
     pip3:
      - python-apt
-     - openshift==0.11.2
```

Comment:
why do we need to remove this?

Reply:
`openshift` was a requirement for the old k8s version `1.2.1` and it depends on `kubernetes==11.0.0`  which overrides the new version needed >=12.0.0